libtiff/libtiff_ctypes.py

```diff
@@ -418,7 +474,7 @@ def add_tags(tag_list):
     TIFFTAG_BITSPERSAMPLE: (ctypes.c_uint16, lambda _d: _d.value),
     TIFFTAG_CLEANFAXDATA: (ctypes.c_uint16, lambda _d: _d.value),
     TIFFTAG_COMPRESSION: (ctypes.c_uint16, lambda _d: _d.value),
-    TIFFTAG_DATATYPE: (ctypes.c_uint16, lambda _d: _d.value),
+    TIFFTAG_DATATYPE: (ctypes.c_uint16, lambda _d: _d.value),  # Obsolete tag replaced by SampleFormat
```

Comment:
What libtiff version are you using? If it is newer than 4.6, I surprised it works because `libtiff/tiff_h_4_6_0.py` does not contain `TIFFTAG_DATATYPE` and I would expect NameError when importing pylibtiff.

Reply:
Isn't it here? https://github.com/pearu/pylibtiff/blob/0537b23eaef62c23b286f7b153d034b4b81c82f3/libtiff/tiff_h_4_6_0.py#L229

I'm using (typically) Ubuntu 22.04, with libtiff 4.3. Do you have a libtiff version which completely dropped that tag? We can handle it too, it's just I didn't know.

The original reason I had commented it out was to fix `test_copy()`. Because `copy()` would copy that tag, and it seems that libtiff would use it to override SampleFormat. That causes test failure because the test explicitly requests to overrides SampleFormat, and sometimes (?!) segfaults. I've now fixed it a better way by just explicitly skipping the copy of this tag.

Reply:
Ah, nvm, I just misread the output of `grep TIFFTAG_DATATYPE ...`. Sorry for the noise.